src/components/hackathon/index.tsx

```diff
@@ -0,0 +1,162 @@
+  },
+];
+
+const faqs: Array<{ question: string, answer: string }> = [
+  {
+    question: "What is the Huff Hackathon?",
+    answer: "The Huff Hackathon is a virtual event where developers can learn about Huff and EVM programming, build projects, and compete for prizes. It's a great opportunity to learn, network, and collaborate with other developers.",
+  },
+  {
+    question: "When and where is the Huff Hackathon?",
+    answer: "The Huff Hackathon is a virtual event that will take place on the Huff Discord server. The event will begin on Friday, October 1st, 2021, and end on Sunday, October 3rd, 2021.",
+  },
+  {
+    question: "How can I participate in the Huff Hackathon?",
+    answer: "To participate in the Huff Hackathon, you must register on our official website. Registration is free and open to developers of all levels and backgrounds.",
+  },
+  {
+    question: "What is the Huff Hackathon?",
+    answer: "The Huff Hackathon is a virtual event where developers can learn about Huff and EVM programming, build projects, and compete for prizes. It's a great opportunity to learn, network, and collaborate with other developers.",
+  },
+  {
+    question: "When and where is the Huff Hackathon?",
+    answer: "The Huff Hackathon is a virtual event that will take place on the Huff Discord server. The event will begin on Friday, October 1st, 2021, and end on Sunday, October 3rd, 2021.",
+  },
+  {
+    question: "How can I participate in the Huff Hackathon?",
+    answer: "To participate in the Huff Hackathon, you must register on our official website. Registration is free and open to developers of all levels and backgrounds.",
+  },
+];
+
+export const Hackathon: React.FC = () => {
+  return (
+    <Wrapper>
+      <Container>
+        <Logo />
+        <div>
+          <h1>World's First Huff Hackathon</h1>
+          <h3 style={{ marginTop: '1vh' }}>Sep 28 - Oct 03, 2023 · Virtual</h3>
+        </div>
+        <Link href="https://jig79bdvw35.typeform.com/to/h3Sjvngb">Register to participate</Link>
+      </Container>
+
+      <Container>
+        <h2>About</h2>
+        <p>
+          Harness the dark arts of EVM with Huff, the cryptic language that
+          separates wizards from muggles.
+          Inscribe your name in the annals of
+          crypto history by joining the Hackathon of the arcane. Embrace the
+          challenge and craft spells of smart contracts that will shape the
+          future!
+          <br />
+          <br />
+          Join now to have fun and win cool prizes: moolah, swag and NFTs!
+        </p>
+      </Container>
+
+      <Container>
+        <h2>Schedule</h2>
+        <Schedule>
+          <tbody>
+            <tr>
+              <td>Event starts</td>
+              <td>Sep 28, 4:00 PM (UTC)</td>
+            </tr>
+            <tr>
+              <td>CTFs &amp; talks</td>
+              <td>Sep 29, TDB (UTC)</td>
```

Comment:
i never heard TDB ive only heard TBD and TBA for "to be determined" or "to be announced"